packages/block-library/src/page-list/edit.js

```diff
@@ -72,11 +72,10 @@ function BlockContent( {
 
 	if ( pages.length === 0 ) {
 		return (
-			<div { ...blockProps }>
-				<Notice status={ 'info' } isDismissible={ false }>
-					{ __( 'Page List: Cannot retrieve Pages.' ) }
-				</Notice>
-			</div>
+			<span
+				className="rich-text"
+				data-rich-text-placeholder="You have no pages yet."
+			></span>
```

Comment:
I tested and just using a simple span is better, no className and no `data-rich-text-placeholder`. Just a span with the text passed through `__`.